xlml/utils/metric.py

```diff
@@ -416,31 +418,53 @@ def get_gce_job_status(benchmark_id: str) -> bigquery.JobStatus:
   context = get_current_context()
   execution_date = context["dag_run"].logical_date
   current_dag = context["dag"]
+  benchmark_id = task_test_config.benchmark_id
 
-  # check setup status to see if provision step is successful
-  setup_task = current_dag.get_task(task_id=f"{benchmark_id}.provision.setup")
-  setup_ti = TaskInstance(setup_task, execution_date)
-  setup_state = setup_ti.current_state()
-  if setup_state == TaskState.SKIPPED.value:
-    logging.info("The setup state is skipped, and the job status is missed.")
-    return bigquery.JobStatus.MISSED
-
-  # check setup status to see if setup step is successful
-  if setup_state == TaskState.FAILED.value:
-    logging.info("The setup state is failed, and the job status is failed.")
+  # GCE SSH method
+  if task_test_config.use_startup_script == False:
```

Comment:
Nit: `if not task_test_config.use_startup_script`?

Reply:
Done.